server/views/partials/leaderboard.pug

```diff
@@ -15,8 +15,13 @@ table.table(cellspacing="0px", width="100%", style="width: 100%;overflow:hidden;
   thead
     
     tr
-      th(scope="row", v-text="'rank'", width="33")
-      th(v-for='(key, j) in Object.keys(teams[0])', v-if="['_id', '__v'].indexOf(key) === -1" v-text='columns[key]', scope="col", width="120")
+      th(v-for='(key, j) in Object.keys(teams[0])',
+        v-if="['_id', '__v'].indexOf(key) === -1",
+        v-html="(key === 'isOnlineHackathon' ? `<ul><li class='avatar'>Online</li><li class='avatar'> | </li><li class='avatar'>In-person</li></ul>` : `<span>${columns[key]}</span>`)",
```

Comment:
This part is appending an extra `undefined` column and I don't know why...

Reply:
I'm using the class 'avatar' because it's set up to line up horizontally.

Reply:
Took a look and there doesn't seem to be an extra column as far as I can tell. Think it might just be the `ul` messing things up a bit.

Reply:
It was the key, 'buildStatus'. This will be different with the next push.